xml/cap_depl_install_minimal.xml

```diff
@@ -305,6 +305,20 @@ Happy Helming!
      is an easy option for a minimal test deployment.
  </para>
 
+   <warning>
+        <title>Warning: Enable hostpath-provisioner on CaaSP</title>
+        <para>
+        CaaSP is configured as a multi-node kubernetes setup with a minimum of one master and two workers. 
+        Enabling hostpath provisioning on CaaSP will use local storage on each of these nodes. 
+        Therefore persistent data stored will also only be available locally on the respective kubernetes nodes. 
+        That directly impacts use cases, where CAP containers could restart on a different kubernetes worker 
```

Comment:
Replace CAP with &scf; for SUSE Cloud Foundry. See section 1.1. The product managers want us to be specific about the various components that make up the SUSE Cloud Application Platform. And to use the correct long product names.